packages/tldraw/src/lib/ui/components/StylePanel/DoubleDropdownPicker.tsx

```diff
@@ -41,14 +41,17 @@ function _DoubleDropdownPicker<T extends string>({
 	valueB,
 	onValueChange,
 }: DoubleDropdownPickerProps<T>) {
+	// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Surprised about this one as well.

Reply:
ah looks like it's because it was `_DoubleDropdownPickerInner` vs. `DoubleDropdownPickerInner` - i'll rename